atd-vze/src/Components/GridTable.js

```diff
@@ -526,15 +523,13 @@ const GridTable = ({
                     moveBack={moveBackPage}
                     pageNumber={page}
                     limit={limit}
-                    totalRecords={totalRecords}
-                    totalPages={totalPages}
                     handleRowClick={handleRowClick}
```

Comment:
One small bug that i noticed which I think already exists is that you can page ahead, update the the row count, and then end up on a page that doesn't exist like if you go from 10 rows per page to 100 rows per page for example. 

Resetting the pagination could be a way to handle this, but I think a spinoff is good since it is reproducible in the current prod editor.

Reply:
great catch! i went ahead and patched that because it seems to have turned out to be a one-liner fix: https://github.com/cityofaustin/atd-vz-data/pull/1507/commits/6d31590c3ef3714b16318cecbc675ff1ae2955e6